flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/module/ModuleManager.java

```diff
@@ -37,6 +42,9 @@
  * Responsible for loading/unloading modules, managing their life cycles, and resolving module objects.
  */
 public class ModuleManager {
+
+	private static final Logger LOG = LoggerFactory.getLogger(CatalogManager.class);
```

Comment:
Should ModuleManager have its own logger?

Reply:
sorry, copy-n-paste mistake